app/daemon/daemon.go

```diff
@@ -281,12 +286,17 @@ func getOSInfo() OSInfo {
 
 func downloadDaemon(serverConf conf.ServerConfig, version string) {
 
+
 	releaseInfo, _ := getReleaseDataForVersion(serverConf.ReleaseAPI, version)
 
 	dlPath, dlName, _ := getDownloadPathAndName(releaseInfo)
 
+	isGettingDaemon = true // flag we are getting the daemon
+
 	fs.DownloadExtract(dlPath, dlName)
```

Comment:
@red010b37 should we check for `err` after `fs.DownloadExtract`?

Reply:
@rex-kwon, no not at this stage as I think i want it to retry and not fatally error. I have found that the github api is rate limited. So need to look at caching that before we sort out the errors.

Reply:
sounds good